### PR TITLE
extensibility updates - create interfaces for journaler and coalescer

### PR DESCRIPTION
**Pull request by ets-mlb**

### CODE REVIEW

File: src/main/java/emissary/output/roller/ICoalescer.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package emissary.output.roller;
+
+import emissary.roll.Rollable;
```
Comment (fbruton) [inline src/main/java/emissary/output/roller/ICoalescer.java]: Remove unused import

Comment (ets-mlb) [reply src/main/java/emissary/output/roller/ICoalescer.java]: done

**Comment by fbruton** [comment]
JournaledCoalescer.java has an unused import as well

File: src/main/java/emissary/output/roller/ICoalescer.java (hunk 1 above)
Comment (ctubbsii) [reply src/main/java/emissary/output/roller/ICoalescer.java]: FYI, my [impsort-maven-plugin](https://github.com/revelc/impsort-maven-plugin) will remove these automatically in the build.

**Pull request merged**